Dockerfile.muo

```diff
@@ -1,30 +0,0 @@
-ARG REGISTRY
```

Comment:
This image is FIPs compliant now which is why we don't have to manually build it, correct?

Reply:
Did we remove any associated pipeline references in ADO if so?  Or were there not any?